### PR TITLE
Tracks 244 & other fixes

### PR DESCRIPTION
**Pull request by SnoogleBunny**
changelog:

* removed contextualMetaData header
* created profile page
* exposed profile page via button on the top right for users to click on
* created profile page dialog for editing personal info
* created a standard for when tables on other pages succeed in making a request but there is genuinely no data available

### CODE REVIEW

File: frontend/src/main/pages/operator/components/ProfileView.tsx [hunk 1]
```diff
@@ -1,12 +1,167 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useSelector } from '../../../../state/utilities/use_selector'
+
+import { Typography, IconButton, Avatar, Paper, Grid, styled, Button, TextField, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@mui/material';
+import { Edit } from '@mui/icons-material';
+
+const roleEnums = {
```
Comment (plasticviking) [inline frontend/src/main/pages/operator/components/ProfileView.tsx]: I'd prefer to see this come from the backend (either statically or in a code table)

Comment (SnoogleBunny) [reply frontend/src/main/pages/operator/components/ProfileView.tsx]: added this to the /me route and doing the string conversion / enumeration from there

File: backend/src/apis/shared/userinfo.ts [hunk 2]
```diff
@@ -24,6 +31,7 @@ const userInfo = {
 			organization: org,
 			name: req.jwtClaims.name,
 			email: req.jwtClaims.email,
+			role: roleEnums[`${req.jwtClaims.roles[0]}`] || 'none',
```
Comment (plasticviking) [inline backend/src/apis/shared/userinfo.ts]: What if they have more than one?

Comment (plasticviking) [inline backend/src/apis/shared/userinfo.ts]: This should be a separate endpoint, probably returning a query result from a role code table

Comment (plasticviking) [reply backend/src/apis/shared/userinfo.ts]: This doesn't handle multiple roles

Comment (SnoogleBunny) [reply backend/src/apis/shared/userinfo.ts]: All good to go on this front. mapped the array to the enumerated values and used join() to display the roles.

**Review by plasticviking — APPROVED** [review]

**Pull request merged**